### DISCUSSION TITLE
Wrong German translation for “has followed”

### DISCUSSION THREAD

**Discussion by jasperhabicht** [community/community#145868] [Mobile]
When a user follows another user or thing, the German translation for this is currently “{user} hat {other user} gefolgt.”, but this is not the correct translation in my opinion. First, folgen is a verb of which the past tense is typically created with sein, not with haben (at least in Standard German and when having the meaning  of following somebody). Second, past tense as it is used here implies that the user followed the other user but does not do this anymore, which I think is not the intended meaning. 

Rather, the translation should be “{user} folgt {other user}.”

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.187.0 (147865513)
</details>

**Comment by stevepopovich** [comment]
Hi @jasperhabicht I have passed this along to the team for consideration, thank you!
